ferrilata-devilline/HelperMethods/HelperMethods.cs

```diff
@@ -18,5 +18,18 @@ public static bool checkMissingPostedPitchFields(AuxPitch NewPitch)
             }
             return false;
         }
+
+        public static bool checkMissingPutPitchFields(Pitch pitch)
+        {
+            bool result = true;
+            if (pitch.Username == null ||
```

Comment:
24:  bool result = true; no need to create result variable, enough to return false in if statement and true otherwise